src/components/grid/grid.ts

```diff
@@ -463,6 +463,20 @@ export class Grid extends Component<Props, SpreadsheetChildEnv> {
     });
   }
 
+  private processSpaceKey(ev) {
+    if (
+      this.env.model.getters.getValidationRuleForCell(this.env.model.getters.getActivePosition())
+        ?.criterion.type === "isBoolean"
```

Comment:
use getter `isCellValidCheckbox`

With your code, you will also change the content of an invalid checkbox cell. "tabouret" will be changed to 'false'. Unless that's the intended behaviour ?